README.md

```diff
@@ -12,41 +12,59 @@ next versions.*
 We offer more details below on how to setup `plz` and run your jobs, but we can
 start by giving you an overview of what `plz` does.
 
-`plz` offers a command-line interface. You can add a `plz.config.json` file
-to the directory where you have your source code. This file contains, among
-other things, the command you run to put your program to work (for instance,
-`python main.py`). Then, you can run commands like `plz run`:
+`plz` offers a command-line interface. You start by adding a `plz.config.json`
+file to the directory where you have your source code. This file contains,
+among other things, the command you run to put your program to work (for
+instance, `python main.py`). Then, you can run commands like `plz run`,
+as shown for this example (that is provided in this repository as well):
 
 ```
-sergio-prodo@sergio:~/plz/examples/python$ plz run
-👌 Capturing the files in /home/sergio-prodo/plz/examples/python
+sergio@sergio:~/plz/examples/pytorch$ plz run
```

Comment:
Is there a short description of what the `pytorch` example is doing?

Reply:
The first paragraph now reads:
"In the directory `plz/examples/pytorch` there's a full-fledged example
for the task of digit recognition using the classic approach of
LeNets and a subset of the well-known MNIST dataset."

What else would you like to see in a short description?